docs/CaliptraIntegrationSpecification.md

```diff
@@ -725,6 +725,145 @@ The following code snippet and schematic diagram illustrate JTAG originating CDC
 * cdc signal reg\_wr\_data  -module dmi\_wrapper -stable
 * cdc signal reg\_wr\_addr  -module dmi\_wrapper -stable
 
+# RDC analysis and constraints
+
+Reset Domain Crossing (RDC) analysis is performed on the Caliptra core IP. The following are the results and recommended constraints for Caliptra integrators using standard RDC analysis EDA tools.
+
+In an unconstrained environment, several RDC violations are anticipated. RDC analysis requires the addition of constraints to identify valid synchronization mechanisms from one reset domain to another.
+
+## Constraining Reset Domains
+
+### Reset Domains
+The table below identifies the major reset domains in Caliptra core IP design.
+
+*Table 20: Reset Definitions (Functional resets are marked in **Bold**)*
```

Comment:
All table numbers defined later in the spec should be incremented to account for new tables here.

Reply:
done, please check